### PR TITLE
Drop duplicated dependencies from `pom.xml`

### PR DESCRIPTION
**Pull request by yevhenii-nadtochii**
This PR addresses [#362](https://github.com/SpineEventEngine/config/issues/362).

In particular, it does the following:

1. Deletes manual applying of `pmd` plugin, which is now applied via our `pmd-settings` script. It eliminates duplicated dependencies on `pmd`.
2. Pulls the latest `config` which contains a [fix](https://github.com/SpineEventEngine/config/pull/368) for the issue. It will hide duplicated dependencies, emitting a warning.
3. Increases amount of memory for JVM metaspace used for Gradle builds.

The library version is bumped to `2.0.0-SNAPSHOT.90`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -154,20 +149,35 @@ all modules and does not describe the project structure per-subproject.
     <artifactId>org.jacoco.ant</artifactId>
     <version>0.8.7</version>
   </dependency>
+  <dependency>
+    <groupId>org.jetbrains.dokka</groupId>
+    <artifactId>dokka-base</artifactId>
+    <version>1.6.20</version>
+  </dependency>
+  <dependency>
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:157]: @sonatype-lift ignore

File: pom.xml [hunk 2]
```diff
@@ -154,20 +149,35 @@ all modules and does not describe the project structure per-subproject.
     <artifactId>org.jacoco.ant</artifactId>
     <version>0.8.7</version>
   </dependency>
+  <dependency>
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:152]: @sonatype-lift ignore

File: pom.xml [hunk 3]
```diff
@@ -154,20 +149,35 @@ all modules and does not describe the project structure per-subproject.
...
+    <groupId>org.jetbrains.dokka</groupId>
+    <artifactId>dokka-base</artifactId>
+    <version>1.6.20</version>
+  </dependency>
+  <dependency>
+    <groupId>org.jetbrains.dokka</groupId>
+    <artifactId>dokka-core</artifactId>
+    <version>1.6.20</version>
+  </dependency>
+  <dependency>
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:162]: @sonatype-lift ignore

**Comment by yevhenii-nadtochii** [comment]
@armiol Please, take a look.

**Review by armiol — APPROVED** [review]
LGTM.

**Pull request merged**